README.md

```diff
@@ -64,7 +92,16 @@ argument. Finally, demonstrate calling `doSomething` with a function.**
 
 Your Answer:
 ```js
-// write code here
+
+function thingToDo(){
+  console.log("Do this!");
+}
+function doSomething(thingToDo){
+  thingToDo();
+}
+
+doSomething(thingToDo);
+
```

Comment:
This is correct! I just want to note that the first function you declared doesn't have to be called `thingToDo`, that's only the argument name. See the solution for an example of this.